### ISSUE TITLE
TypeError: normalizeAssertion is not a function or its return value is not iterable

### ISSUE THREAD

**Issue by thomkrupa** [GoogleChrome/lighthouse-ci#64]
Hey,

After upgrading `@lhci/cli` to `3.1.0` (from `0.3.0-alpha.0`) I can no longer use a custom lighthouserc config.

Here is my `lighthouserc.json`:

```json
{
  "ci": {
    "assert": {
      "preset": "lighthouse:recommended",
      "assertions": {
        "categories.pwa": "off"
      }
    }
  }
}
```

and logs:

```
Run lhci autorun --config=./lighthouserc.json
✅  .lighthouseci/ directory writable
✅  Configuration file found
Healthcheck passed!

Automatically determined ./public as `staticDistDir`.
Set it explicitly in lighthouserc.json if incorrect.

Started a web server on port 38257...
Running Lighthouse 3 time(s) on http://localhost:38257/index.html
Run #1...done.
Run #2...done.
Run #3...done.
Done running Lighthouse!

TypeError: normalizeAssertion is not a function or its return value is not iterable
    at getAllFilteredAssertionResults (/opt/hostedtoolcache/node/10.17.0/x64/lib/node_modules/@lhci/cli/node_modules/@lhci/utils/src/assertions.js:390:39)
    at getAllAssertionResults (/opt/hostedtoolcache/node/10.17.0/x64/lib/node_modules/@lhci/cli/node_modules/@lhci/utils/src/assertions.js:440:23)
    at Object.runCommand (/opt/hostedtoolcache/node/10.17.0/x64/lib/node_modules/@lhci/cli/src/assert/assert.js:54:22)
    at run (/opt/hostedtoolcache/node/10.17.0/x64/lib/node_modules/@lhci/cli/src/cli.js:87:23)
    at Object.<anonymous> (/opt/hostedtoolcache/node/10.17.0/x64/lib/node_modules/@lhci/cli/src/cli.js:118:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)

    at Function.Module._load (internal/modules/cjs/loader.js:585:3)assert command failed. Exiting with status code 1.
##[error]Process completed with exit code 1.
```

I'm not sure if I missed something during the update or is it a bug?

**Comment by patrickhulce** [comment]
Thanks so much for filing @thomkrupa! This is a unexpected bug with how config is handled between 0.3.0-alpha and 0.3.0, fix on the way :)

**Comment by patrickhulce** [comment]
fixed by 1aaeab26aef5df76ff735bb6778f82b1ffdf54ec and published in 0.3.2

**Comment by patrickhulce** [comment]
Oh I just realized @thomkrupa that you might have done this to attempt to turn off all the PWA audits from the preset. Right now it would only disable an assertion on the category *score* but not the individual audits. There's a proposal to add a `lighthouse:no-pwa` preset that it seems like you might support?

https://github.com/GoogleChrome/lighthouse-ci/issues/51#issuecomment-552118992
